plugins/network-generator/src/main/kotlin/dev/icerock/moko/network/SpecInfo.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2021 IceRock MAG Inc. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package dev.icerock.moko.network
+
+import org.openapitools.generator.gradle.plugin.tasks.GenerateTask
+import java.io.File
+
+open class SpecInfo(val name: String) {
+    var inputSpec: File? = null
+    var packageName: String? = null
+    var isInternal = true
+    var isOpen = false
+    var configureTask: (GenerateTask.() -> Unit)? = null
```

Comment:
better to do this var `internal` and add `fun configureTask(block: GenerateTask.() -> Unit) { configureTask = block }` to better usage